### PR TITLE
Gitignore notes and word-based diffing

### PR DESCRIPTION
**Pull request by junghans**
- Added some notes about `git://` and ssh proxy
- Added a note about `**` pattern in gitignore section
- Mention `--color-words` option in diff section, which is extremely useful for LaTeX files.

### CODE REVIEW

File: _episodes/04-changes.md [hunk 1]
```diff
@@ -474,6 +474,13 @@ Date:   Thu Aug 22 09:51:46 2013 -0400
 ~~~
 {: .output}
 
+> ## Word-based diffing
+>
+> Sometimes, e.g. in the case of the LaTeX documment a line-wise
```
Comment (iglpdc) [inline _episodes/04-changes.md]: documment -> document

Comment (iglpdc) [inline _episodes/04-changes.md]: There is a missing comma after "document":

"Sometimes, e.g. in the case of a LaTeX document, ..."

but I think it may even better not to mention the LaTeX example: most of our learners do not use it and this may make them think that the color diff is not useful for them.

File: _episodes/04-changes.md [hunk 2]
```diff
@@ -474,6 +474,13 @@ Date:   Thu Aug 22 09:51:46 2013 -0400
...
+> diff is too coarse. That is where the `--color-words` option of
```
Comment (iglpdc) [inline _episodes/04-changes.md]: I would rephrase it to explain a bit what color-words does. From the docs: "Highlights the changed words using colors". What do you think of changing your sentence "That is where ... " to include something similar?

File: _episodes/02-setup.md [hunk 3]
```diff
@@ -98,6 +98,8 @@ same commands to choose another editor or update your email address.
 > $ git config --global --unset https.proxy
 > ~~~
 > {: .bash}
+> In addition git supports the SSH protocol, which needs no proxy configuration, and
```
Comment (iglpdc) [inline _episodes/02-setup.md]: I'd prefer not to include this change. I think it's useful information for a more advanced crowd, but it may be confussing for a novice. The only reason the proxy callout is here is because it may be needed in some workshops with a weird network configuration.

File: _episodes/07-github.md [hunk 4]
```diff
@@ -144,6 +144,7 @@ Branch master set up to track remote branch master from origin.
 > $ git config --global --unset https.proxy
 > ~~~
 > {: .bash}
+> Please remember that for using SSH as connection, proxy setup is not needed.
```
Comment (iglpdc) [inline _episodes/07-github.md]: Something as above. I think mentioning details of SSH won't be useful and may be confussing for most of our learners. We have a few links to set up properly SSH in this episode and I think this is enough.

**Comment by junghans** [comment]
@iglpdc: Done!

**Comment by iglpdc** [comment]
Thanks a lot @junghans !
